src/schemas.ts

```diff
@@ -49,4 +63,34 @@ export interface SchemaEmail extends SchemaRecord {
     sentAt: string
     attachments?: SchemaEmailAttachment[]
     threadId?: string
+}
+
+export enum SchemaYoutubeActivityType {
```

Comment:
This isn't part of our schemas, so shouldn't be in this file.

Rename to `YoutubeActivityType` and put in a `providers/google/interfaces.ts` file as it's specific to the Google API.

Reply:
> This isn't part of our schemas, so shouldn't be in this file.
> 
> Rename to `YoutubeActivityType` and put in a `providers/google/interfaces.ts` file as it's specific to the Google API.

Fixed